.env

```diff
@@ -48,6 +53,16 @@ LTI1P3_REGISTRATION_ID=demo
 ###< LTI ###
 
 ###> blackfire ###
-BLACKFIRE_SERVER_ID=<your_backfire_id>
-BLACKFIRE_SERVER_TOKEN=<your_backfire_secret>
+BLACKFIRE_ENABLED=false
```

Comment:
Would be good to have some sort of presentation/training about blackfire in the future. 
Have no exposure to it at all

Reply:
Good idea, I'll figure out something :)